CHANGES.md

```diff
@@ -10,13 +10,16 @@
 
 * ancestral, refine: Explicitly specify how the root and ambiguous states are handled during sequence reconstruction and mutation counting. [#1690][] (@rneher)
 * titers: Fix type errors in code associated with cross-validation of models. [#1688][] (@huddlej)
-* Add help text to clarify difference in behavior between options that override defaults (e.g. `--metadata-delimiters`) vs. options that extend existing defaults (e.g. `--expected-date-formats`). [#1705][] (@victorlin)
```

Comment:
just curious about the reasoning behind removing this changelog entry?

Reply:
89fbda200f4b5b4f423c0add9432e4c8b34683fb